server/routes/api/columns.js

```diff
@@ -46,20 +46,22 @@ router.post(
 // @access private
 router.put(
   "/:id",
-  [auth, columnValidationRules(), validate],
+  [auth, validate],
```

Comment:
Bahaha I left the columnValidationRules() back in my hotfix because it required the "name" / title property of the column. I think it's fine leaving it out since there's no real validation other than checking the name.

Reply:
Yeah hard agree on what you said and prob should change the columnValidatoinRules in the future lol